nmstatectl/prettystate.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from collections import OrderedDict
+import json
+from operator import itemgetter
+
+
+import yaml
+
+
+class PrettyState(object):
+    def __init__(self, state):
+
+        yaml.add_representer(OrderedDict, represent_ordereddict)
+        self.state = order_state(state)
+
+    @property
+    def yaml(self):
+        return yaml.dump(self.state, default_flow_style=False)
+
+    @property
+    def json(self):
+        return json.dumps(self.state, indent=4, separators=(',', ': '))
+
+
+def represent_ordereddict(dumper, data):
+    """
+    Represent OrderedDict as regular dictionary
+
+    Source: https://stackoverflow.com/questions/16782112/can-pyyaml-dump-dict-items-in-non-alphabetical-order
+    """  # noqa: E501
```

Comment:
I prefer you break the line over breaking format rules.

Reply:
60 votes is not that convincing, but if you prefer it like this lets go with the exception.

Reply:
All but one answer are for keeping the URL this way.